Utilities/bootstrap

```diff
@@ -127,11 +149,12 @@ if platform.system() == 'Darwin':
         ["xcrun", "--sdk", "macosx", "--show-sdk-path"],
         universal_newlines=True).strip()
 
-if platform.system() == 'Linux':
-    g_shared_lib_ext = ".so"
-else:
+elif platform.system() == 'Darwin':
```

Comment:
You have if Darwin else if Darwin; need to collapse these two

Reply:
Oops; I meant to make that an `if`.